### PR TITLE
[docs] List multiple identity providers, link to config guides

### PR DESCRIPTION
**Pull request by keith-kurak**
# Why
We are adding additional identity providers to SSO, so need to expand this out.

I think some users miss that only specific providers are supported (based on questions we get). Hopefully a table helps make it more obvious.

Additionally, a goal here is to telegraph what are the next steps, suggesting how a requestor can get ready (by starting the configuration), or at least what we're going to ask them when they contact us. I also want to move the configuration guides into a more permanent place.

# How
Added a table with IdP's and a link to their config guide (pending push on expo/fyi)

# Test Plan
Check the docs

# Checklist

- [ ] Documentation is up to date to reflect these changes (eg: https://docs.expo.dev and README.md).
- [ ] Conforms with the [Documentation Writing Style Guide](https://github.com/expo/expo/blob/main/guides/Expo%20Documentation%20Writing%20Style%20Guide.md)
- [ ] This diff will work correctly for `npx expo prebuild` & EAS Build (eg: updated a module plugin).

### CODE REVIEW

**Review by lzkb — APPROVED** [review]

File: docs/pages/accounts/sso.mdx [hunk 1]
```diff
@@ -6,9 +6,18 @@ description: Learn how your enterprise organization can use your Identity Provid
 import { Step } from '~/ui/components/Step';
 import { Terminal } from '~/ui/components/Snippet';
 
-SSO is available for [Enterprise Plan](https://expo.dev/pricing) customers. To get started, contact us and we'll help you set it up for your organization.
+SSO is available for [Enterprise Plan](https://expo.dev/pricing) customers. To get started, [contact us](https://expo.dev/contact) and we'll help you set it up for your organization. You can also start preparing your identity provider for Expo SSO by following the configuration guides below.
 
-> The Expo SSO system currently formally supports [Okta](https://www.okta.com/), but we implement the [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html) specification and are working to verify additional compatible identity providers. If you use another identity provider and are interested in SSO, let us know.
+## Identity provider support
+
+Currently, Expo SSO supports the following identity providers:
```
Comment (amandeepmittal) [inline docs/pages/accounts/sso.mdx:13]:
```suggestion
Expo SSO supports the following identity providers:
```

- The word "Currently", is already implicit here.

File: docs/pages/accounts/sso.mdx [hunk 2]
```diff
@@ -6,9 +6,18 @@ description: Learn how your enterprise organization can use your Identity Provid
...
+
+| Identity providers                                                                       | Resources                                                       |
+| ---------------------------------------------------------------------------------------- | --------------------------------------------------------------- |
+| [Okta](https://www.okta.com/)                                                            | [Expo configuration guide](https://expo.fyi/sso-setup-okta)     |
+| [OneLogin](https://www.onelogin.com/)                                                    | [Expo configuration guide](https://expo.fyi/sso-setup-onelogin) |
```
Comment (amandeepmittal) [inline docs/pages/accounts/sso.mdx:18]:
```suggestion
| [Okta](https://www.okta.com/)                                                            | [Configuration guide](https://expo.fyi/sso-setup-okta)     |
| [OneLogin](https://www.onelogin.com/)                                                    | [Configuration guide](https://expo.fyi/sso-setup-onelogin) |
```

File: docs/pages/accounts/sso.mdx [hunk 3]
```diff
@@ -6,9 +6,18 @@ description: Learn how your enterprise organization can use your Identity Provid
...
+
+We implement the [OpenID Connect Discovery 1.0](https://openid.net/specs/openid-connect-discovery-1_0.html) specification and are working to verify additional compatible identity providers. If you use another identity provider and are interested in SSO, let us know.
```
Comment (amandeepmittal) [inline docs/pages/accounts/sso.mdx:20]: We should add a link on how to contact us for "let us know".

File: docs/pages/accounts/sso.mdx [hunk 4]
```diff
@@ -6,9 +6,18 @@ description: Learn how your enterprise organization can use your Identity Provid
 import { Step } from '~/ui/components/Step';
 import { Terminal } from '~/ui/components/Snippet';
 
-SSO is available for [Enterprise Plan](https://expo.dev/pricing) customers. To get started, contact us and we'll help you set it up for your organization.
+SSO is available for [Enterprise Plan](https://expo.dev/pricing) customers. To get started, [contact us](https://expo.dev/contact) and we'll help you set it up for your organization. You can also start preparing your identity provider for Expo SSO by following the configuration guides below.
```
Comment (amandeepmittal) [inline docs/pages/accounts/sso.mdx:9]:
```suggestion
SSO is available for [Enterprise plan](https://expo.dev/pricing) customers. To get started, [contact us](https://expo.dev/contact) and we'll help you set it up for your organization. You can also start preparing your identity provider for Expo SSO by following the configuration guides below.
```

**Review by amandeepmittal — APPROVED** [review]

**Pull request merged**